qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -14,11 +14,12 @@
 """This module contains functions to construct many-body observables whose expectation
 values can be used to simulate molecular properties.
 """
+import numpy as np
+
```

Comment:
Why was this moved to the top ? was there a pylint issue or something ?

Reply:
I run `isort` to sort the imports.